### ISSUE TITLE
Bandwidth Errors continue with 7.3.4

### ISSUE THREAD

**Issue by EricFlight** [derekhe/msfs2020-map-enhancement#484]
**Describe the bug**

7.3.4. started doing the same thing  as the previous versions. The Cache size goes from the normal size to showing 0 again. There's something wrong with the cache. I tried to fix by doing what I did before (delete old make new cache/enhanced versus normal/run it as administrator/double check Google is not blocking me by checking maps.google.com/try a different ssd for cache storage/etc...). I also did a reinstall of Windows 11 and fresh install of the simulator (had to do it not because of the Map Enhancement but wanted you to know it rules out all possibilities of it being a Win 11/MSFS problem). Something is still wrong with how the cache works.

If I turn the cache off it works fine.

If I go to settings->Cache options->Choose Cache File and then choose the cache file (after it shows 0 for Cache Size), the cache size pops back up normal for a while and it works (still shows a bounce back to 0 for a cache size for a second after a while). It eventually goes back to having 0 for a cache size for a while after that and then it is too slow to download photogrammetry and turns it off. It's like it forgets where the cache file is

**MSFS version:**
 - Steam
 - v1.34.16.0

**Mod version:**
- Mod version 7.3.4 (Pro)

**Device ID**
- Device ID NRFD3ZHV2RY47DE48AX9JNVXNJAY3G9FTJQDCMQA91RZH8A3H5Y0

**Screenshots**

Please provide me screenshots for the mod and msfs2020 if possible

**Additional context**

**Comment by derekhe** [comment]
thanks for reporting, I will have a look

Eric Flight ***@***.***>于2023年10月23日 周一12:17写道：

**Comment by EricFlight** [comment]
Definitely something wrong with the cache creation process as if you turn off cache it works fine. I've tested it multiple times and for hours.

**Comment by derekhe** [comment]
Thanks for the feedback I will investigate





------------------ Original ------------------
From: Eric Flight ***@***.***&gt;
Date: Tue,Oct 24,2023 11:30 AM
To: derekhe/msfs2020-map-enhancement ***@***.***&gt;
Cc: hesicong ***@***.***&gt;, Comment ***@***.***&gt;
Subject: Re: [derekhe/msfs2020-map-enhancement] Bandwidth Errors continue with7.3.4 (Issue #484)





 
Definitely something wrong with the cache creation process as if you turn off cache it works fine. I've tested it multiple times and for hours.

**Comment by EricFlight** [comment]
Another observation..I think it only starts kicking me off when the cache file size is around 6GB and greater

**Comment by EricFlight** [comment]
Could using this program interfere with the cache? https://www.wagnardsoft.com/ISLCw

I'm trying it wihtout using the ISLC software but wanted to ask as I was running it earlier. I'll let you know if it gets better

**Comment by buehner** [comment]
I have the same/similar problems, so I dont think that other software interferes here (as I am not using it).

My problem is: I am using the map enhancement (pro licence) with "google server 2" most of the time (as you never get blocked by google on this one). Always with the "High LOD" enabled. I have latest hardware and best bandwidth you can imagine. I usually fly IFR short/long haul.

More or less "randomly" I get exactly those bandwidth errors as Eric recorded them in this video: https://www.twitch.tv/videos/1902366906

I never had these problems in the past. This started a few weeks ago.

As this is so annoying, I have to stop using Map enhancement now. I really like it, but it does not make sense as long as this is not fixed. Let me know if I can help debugging it.

**Comment by EricFlight** [comment]
> Could using this program interfere with the cache? https://www.wagnardsoft.com/ISLCw
> 
> I'm trying it without using the ISLC software now but wanted to ask as I was running it earlier. I'll let you know if it gets better

The software was not a problem. Happens either way

**Comment by Olivier78370** [comment]
Config : 7.4 Pro/ Google server 2 / High LOD.
As others on this topic, I have a trouble with Ehancement cache
Definitely something wrong with the cache creation process as if you turn off cache it works fine.

**Comment by EricFlight** [comment]
Any updates? Look forward to using the cache again as downloading without cache is fine but lots of data and it loads slower than having a cache file

**Comment by Olivier78370** [comment]
New test tonight with cache and same config : 7.4 Pro/ Google server 2 / High LOD. 
Seams good with no problems with cache. 
All good now

**Comment by EricFlight** [comment]
> New test tonight with cache and same config : 7.4 Pro/ Google server 2 / High LOD. Seams good with no problems with cache. All good now

Where did you download version 7.4?

**Comment by Olivier78370** [comment]
7.3.4 of course!

**Comment by Olivier78370** [comment]
7.3.4 of course!

**Comment by EricFlight** [comment]
Was working for a bit with cache but just starting saying data too slow and turned off photogrammetry. Works fine without cache. Cache needs to be fixed on how it loads and stores data

**Comment by EricFlight** [comment]
Still get kicked off photogrammetry when cache is enabled. Not as often as before but still kicks data off. If I don't use the cache, works fine

**Comment by derekhe** [comment]
I'm investigating the cache issue.

**Comment by derekhe** [comment]
I have a fix to turn the cache off automatically if the read/write exceeds 150ms which may cause the issue. Please have a try.

**Comment by EricFlight** [comment]
I'll try it once I'm done video editing

Eric

**Comment by EricFlight** [comment]
Before I try it, is there a way to turn the cache back on automatically if
the correct conditions are restored?

**Comment by EricFlight** [comment]
> I have a fix to turn the cache off automatically if the read/write exceeds 150ms which may cause the issue. Please have a try the latest 7.4.0.

Still kicking me off even with that feature on 7.4.0

Eric

**Comment by Olivier78370** [comment]
For me, with 7.4 new version, cache and google server 1 are working fine now. No more problem.

**Comment by EricFlight** [comment]
(Again posted here just in case the other thread gets closed) (7.4.1 beta) Ran it again today offstream. I can confirm it stops working when my license info somehow changes from being fine to No License whiule using it. This is the only problem I think which is why it blocks me from accessing any server. Have any suggestions for my license to keep working and loaded in Map Enhancement? (is good through December 23, 2023 and had no problems on earlier versions)

**Comment by derekhe** [comment]
@EricFlight you can have a try the network setting to "Normal" which has lower concurrent connections to server, maybe better for your network.

**Comment by EricFlight** [comment]
> @EricFlight you can have a try the network setting to "Normal" which has lower concurrent connections to server, maybe better for your network.

It's the pro license that disappears and then It disconnects everything and blocks me from accessing any server in the map enhancement. It is not switching to network...I opened up a new issue to address this. Why is it license disappearing in the Map Enhancement when using it?
